examples/src/main/python/hbase_inputformat.py

```diff
@@ -16,6 +16,7 @@
 #
 
 import sys
+import ast
 
 from pyspark import SparkContext
 
```

Comment:
Great idea. I will modify the sample data to show multiple records in a columnFamily and columnFamily:qualifier